src/scikit_package/cli/gh.py

```diff
@@ -0,0 +1,60 @@
+def _get_broadcast_repos_dict(url_file_path=None):
+    """Load the recognizable broadcast repository information from files
+    in the given path or GitHub repo.
+
+    If ``url_file_path`` is not None, first try to treat it as an URL, and if
+    that fails, treat it as a directory path. If both fail, raise
+    ``ValueError``. If ``url_file_path`` is None, the broadcast_url_file_path
+    set in ``~/.skpkgrc`` will be used. If it is not found, use the current
```

Comment:
Is this the wrong way around? Desired behavior is to Only use .skpkgrc of the files are not found in cwd and no url is passed at runtime